include/aws/http/connection.h

```diff
@@ -206,6 +236,8 @@ struct aws_http_client_connection_options {
 
     /**
      * Optional.
+     * The initial connection flow-control window size for HTTP/1 connection.
+     * Ignored by HTTP/2 connection, since the initial connection flow-control window in HTTP/2 is not configurable.
      * A default size is set by AWS_HTTP_CLIENT_CONNECTION_OPTIONS_INIT.
      */
     size_t initial_window_size;
```

Comment:
We probably want to rename this. When we resolve the WINDOW HTTP/1 & 2 problems.

Reply:
👍yeah, we'll worry about this when we resolve the other problems